{
  "name": "catch-to-llm",
  "version": "0.1.0",
  "description": "GenAI Exception Debugging Developer Tools",
  "main": "dist/src/server.js",
  "scripts": {
    "start": "NODE_ENV=production node ./dist/src/server.js",
    "dev": "concurrently \"npm run dev:server\" \"npm run dev:webapp\"",
    "dev:server": "NODE_ENV=development concurrently \"tsc -w -p tsconfig.json\" \"nodemon ./dist/src/server.js\"",
    "dev:webapp": "webpack serve --config webpack.config.webapp.js --mode development",
    "build": "npm run build:server && npm run build:webapp",
    "build:server": "tsc -p tsconfig.json",
    "build:webapp": "webpack --config webpack.config.webapp.js --mode production",
    "test": "jest"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/rconard/catch-to-llm.git"
  },
  "author": "Russell Conard",
  "bugs": {
    "url": "https://github.com/rconard/catch-to-llm/issues"
  },
  "homepage": "https://github.com/rconard/catch-to-llm#readme",
  "dependencies": {
    "@typescript-eslint/typescript-estree": "^7.16.0",
    "acorn": "^8.12.1",
    "acorn-walk": "^8.3.3",
    "cors": "^2.8.5",
    "detect-node": "^2.1.0",
    "dotenv": "^16.4.5",
    "error-stack-parser": "^2.1.4",
    "express": "^4.19.2",
    "react": "^18.3.1",
    "react-dom": "^18.3.1",
    "react-router-dom": "^6.25.0",
    "source-map": "^0.7.4",
    "uuid": "^10.0.0",
    "winston": "^3.13.0"
  },
  "devDependencies": {
    "@types/express": "^4.17.21",
    "@types/jest": "^29.5.12",
    "@types/node": "^20.14.10",
    "@types/react": "^18.3.3",
    "@types/react-dom": "^18.3.0",
    "@types/react-router-dom": "^5.3.3",
    "@types/supertest": "^6.0.2",
    "concurrently": "^8.2.2",
    "eslint": "^9.6.0",
    "html-webpack-plugin": "^5.6.0",
    "jest": "^29.7.0",
    "nodemon": "^3.1.4",
    "supertest": "^7.0.0",
    "ts-jest": "^29.2.0",
    "ts-loader": "^9.5.1",
    "ts-node": "^10.9.2",
    "typescript": "^5.5.3",
    "webpack": "^5.93.0",
    "webpack-cli": "^5.1.4",
    "webpack-dev-server": "^5.0.4"
  }
}
